pkgs/development/libraries/arrow-cpp/default.nix

```diff
@@ -117,6 +125,7 @@ stdenv.mkDerivation rec {
     rapidjson
     re2
     snappy
+    sqlite
```

Comment:
Is that one required unconditionally?

Reply:
No, just for Flight SQL I believe. I'll fix it u.

Reply:
Fixed, it's only used for testing, so I moved it to `installCheckInputs`.